boot/pod/src/boot/repl.clj

```diff
@@ -58,3 +58,8 @@
   [{:keys [bind port init-ns middleware handler] :as options}]
   (let [opts (->> options setup-nrepl-env!)]
     (@start-server opts)))
+
+(defn launch-bare-repl
+  [opts]
```

Comment:
Should we document the options? https://clojure.github.io/clojure/clojure.main-api.html#clojure.main/repl